.travis.yml

```diff
@@ -8,3 +8,9 @@ env:
   - STACK=cedar-14
 script:
   - bin/docker-build.sh $STACK
+  - |
```

Comment:
I have a mild preference that this be written as a shell script in a file, rather than inline in `.travis.yml`, as it makes the travis file a bit easier to read if the task has a name rather than just a bolus of code.